server/lib/routes/get-openid-configuration.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+var config = require('../configuration');
+
+var issuer = config.get('public_url');
```

Comment:
Unless one variable depends on another, might as well alphabetize the variable names too.